README.md

```diff
@@ -8,85 +8,119 @@ Current state of the software:
 
-The `build/run_mcmc` executable can be used to perform all the necessary analysis, but it needs some command-line arguments to be passed.
-To perform your first run of the library right out of the box, you can call the following script from the command line:
+```shell
+pacman -Sy mingw-w64-x86_64-make
+pacman -Sy mingw-w64-x86_64-cmake
+pacman -Sy mingw-w64-x86_64-pkg-config
+```
+
+### Configure RTools42 / RTools43
+
+These two versions of RTools toolchain are quite similar. Of course, RTools43 offers newer version of the packages, but in both cases the installation and configuration is identical.
+Download either the [RTools42 installer](https://cran.r-project.org/bin/windows/Rtools/rtools42/files/rtools42-5355-5357.exe) or the [RTools43 installer](https://cran.r-project.org/bin/windows/Rtools/rtools43/files/rtools43-5550-5548.exe) and complete the prompts to install.
+Next, add the following lines to your `PATH`:
 
 ```shell
-examples/tutorial/run.sh
+# RTools42
+C:\rtools42\x86_64-w64-mingw32.static.posix\bin
+C:\rtools42\ucrt64\bin
+C:\rtools42\usr\bin
+
+# RTools43
+C:\rtools43\x86_64-w64-mingw32.static.posix\bin
+C:\rtools43\ucrt64\bin
+C:\rtools43\usr\bin
 ```
 
-This is an example script that runs said executable by passing appropriate arguments to it.
-In order to use your custom datasets, algorithm settings, and prior specifications, you can create a copy of the above script and change the arguments as appropriate.
-Please refer to the [documentation](#Documentation) for more information.
+The C++ compiler is now available on your system. All the other dependencies can be installed by typing the following commands into one of the shells installed with RTools (e.g. lauch the `C:\rtools42\msys.exe` or `C:\rtools43\msys.exe` file)
+
+```shell
+pacman -Sy mingw-w64-ucrt-x86_64-make
+pacman -Sy mingw-w64-ucrt-x86_64-cmake
+pacman -Sy mingw-w64-ucrt-x86_64-pkgconf
+```
 
-## For developers
+## Build `bayesmix`
 
-We heavily depend on the `protobuf` library to move and store structured data.
-The `CMakeLists.txt` file is set up to install such library if it does not find it in the computer.
-However any call to `make clean` will uninstall it, causing a huge waste of time... so make sure to install it manually beforehand!
-If you're using a Linux machine, you can do so as follows:
+> **Notes for Windows**:
+>
+> Use the Git BASH shell available with Git for Windows to execute these commands. If `PATH` environment variable has been configured correctly, all requirements will be satisfied.
+>
+> In order for `bayesmix` to be properly linked to Intel's TBB library, the absolute path to `tbb` must be added to the User `PATH` variable. This is done automatically during build but to make this change effective user need to close and open a new Git BASH shell.
```

Comment:
I think this section should be moved to the "install for windows users" Markdown file and maybe mentioned here via a hyperlink

Reply:
Thanks a lot! I'll move all the details into an INSTALL.md file.